### PR TITLE
loading network with admin crendentials

### PR DESCRIPTION
**Pull request by hmunfru**
@jesuspg 
@jframos 
https://jira.tid.es/browse/CLAUDIA-5016
https://jira.tid.es/browse/CLAUDIA-5017

### CODE REVIEW

File: core/src/main/java/com/telefonica/euro_iaas/paasmanager/claudia/impl/OpenstackNetworkClientImpl.java [hunk 1]
```diff
@@ -513,18 +514,22 @@ public NetworkInstance loadNetwork(ClaudiaData claudiaData, NetworkInstance netw
...
             List<NetworkInstance> loadAllNetworks = this.loadAllNetwork(claudiaData, region);
             for (NetworkInstance net : loadAllNetworks) {
-                if (net.getNetworkName().equals(network.getNetworkName()) &&
-                    net.getTenantId().equals(claudiaData.getVdc())) {
-                    return net.getIdNetwork();
+                if (net.getNetworkName().equals(network.getNetworkName())) {
+                    if (net.getShared()) {
+                        networkId = net.getIdNetwork();
+                    } else if (net.getTenantId().equals(claudiaData.getVdc())){
+                        networkId =  net.getIdNetwork();
```
Comment (jframos) [inline core/src/main/java/com/telefonica/euro_iaas/paasmanager/claudia/impl/OpenstackNetworkClientImpl.java]: You are executing the same code as the previous conditional sentence _(if)_. Is that correct? Is it a **OR** clause?

File: core/src/main/java/com/telefonica/euro_iaas/paasmanager/manager/impl/NetworkInstanceManagerImpl.java [hunk 2]
```diff
@@ -179,6 +184,17 @@ private void restoreNetwork(ClaudiaData claudiaData, NetworkInstance networkInst
         networkClient.destroyNetwork(claudiaData, networkInstance, region);
     }
 
+    private String getAdminTenantId (ClaudiaData data) {
+        try {
+            OpenStackAccess openStackAccess = this.openStackRegion.getTokenAdmin();
+            return openStackAccess.getTenantId();
+        } catch (Exception e) {
+            return null;
+
```
Comment (jesuspg) [inline core/src/main/java/com/telefonica/euro_iaas/paasmanager/manager/impl/NetworkInstanceManagerImpl.java]: add error trace

**Comment by jframos** [comment]
LGTM

**Comment by jesuspg** [comment]
LGTM

**Pull request merged**